lib/virtualGraphs.js

```diff
@@ -18,12 +19,14 @@ const add = (conn, name, mappings, options) => {
       name,
       mappings,
       options,
+      db,
     }),
     headers,
   }).then(httpBody);
 };
 
-const update = (conn, name, mappings, options) => {
+const update = (conn, name, mappings, options, meta) => {
+  const { db } = meta;
```

Comment:
Same suggestion as above:

```suggestion
const update = (conn, name, mappings, options, meta = {}) => {
  const { db } = meta;
```